spec/requests/admin/nested_sidebar_spec.rb

```diff
@@ -27,17 +27,15 @@
   end
 
   describe "profile admin feature flag" do
-    it "does not show the option in the sidebar when the feature flag is disabled" do
+    it "shows the option in the sidebar even when the feature flag is disabled" do
```

Comment:
I like this add.  Emphasis on _even_

Reply:
this will flatten shortly in the next PR